ui/pages/swaps/exchange-rate-display/exchange-rate-display.stories.js

```diff
@@ -68,41 +70,11 @@ export const WhiteOnBlue = (args) => {
   );
 };
 
-WhiteOnBlue.argTypes = {
-  primaryTokenValue: {
-    control: {
-      type: 'text',
-    },
-    defaultValue: '2000000000000000000',
-  },
-  primaryTokenDecimals: {
-    control: {
-      type: 'number',
-    },
-    defaultValue: 18,
-  },
-  primaryTokenSymbol: {
-    control: {
-      type: 'text',
-    },
-    defaultValue: 'ETH',
-  },
-  secondaryTokenValue: {
-    control: {
-      type: 'text',
-    },
-    defaultValue: '200000000000000000',
-  },
-  secondaryTokenDecimals: {
-    control: {
-      type: 'number',
-    },
-    defaultValue: 18,
-  },
-  secondaryTokenSymbol: {
-    control: {
-      type: 'text',
-    },
-    defaultValue: 'ABC',
-  },
+WhiteOnBlue.args = {
+  primaryTokenValue: '2000000000000000000',
+  primaryTokenDecimals: 18,
+  primaryTokenSymbol: 'ETH',
+  secondaryTokenValue: '200000000000000000',
+  secondaryTokenDecimals: 8,
+  secondaryTokenSymbol: 'ABC',
```

Comment:
Exchange rate display story works same as before. Updating arg name to reflect prop. Updating deprecated defaultValue syntax.

### Before
<img width="1440" alt="Screenshot 2022-12-11 at 8 37 41 PM" src="https://user-images.githubusercontent.com/8112138/206962184-4dbf5c9c-0144-4664-80bf-01b81d600b85.png">
<img width="1440" alt="Screenshot 2022-12-11 at 8 38 44 PM" src="https://user-images.githubusercontent.com/8112138/206962208-68f2cd6e-616d-414e-be48-35f6228da159.png">

### After
<img width="1440" alt="Screenshot 2022-12-11 at 8 37 58 PM" src="https://user-images.githubusercontent.com/8112138/206962241-de42a441-4703-4125-a701-c62390d834e3.png">
<img width="1440" alt="Screenshot 2022-12-11 at 8 38 22 PM" src="https://user-images.githubusercontent.com/8112138/206962242-f36581eb-bc8d-4e9e-98de-2dff36e43ffa.png">